conf/dev.env

```diff
@@ -0,0 +1,4 @@
+ST2_USER=sam
```

Comment:
Let's stick with usual `st2admin`, give a diff pass may be. Yes I see defaults are "usual", but in practice everyone running this will get "unusual" and will be fighting muscle memory.